packages/react-dialog/src/components/Dialog/renderDialog.tsx

```diff
@@ -1,13 +1,19 @@
 import * as React from 'react';
 import { getSlots } from '@fluentui/react-utilities';
 import type { DialogState, DialogSlots } from './Dialog.types';
+import { Portal } from '@fluentui/react-portal';
 
 /**
  * Render the final JSX of Dialog
  */
 export const renderDialog_unstable = (state: DialogState) => {
   const { slots, slotProps } = getSlots<DialogSlots>(state);
+  const defaultOverlay = state.type !== 'non-modal' && <div aria-hidden="true" className={state.overlayClassName} />;
```

Comment:
Do we also plan to add `aria-hidden` to all DOM subtrees except of the currently open dialog? It was necessary few months ago, not sure if screen reader advanced in this recently and actually treat aria-modal correctly. What would you advise @smhigley , @ling1726

Reply:
`useModalAttributes` is used, so tabster will add `aria-hidden` everywhere